addons/mail/models/mail_thread.py

```diff
@@ -2505,19 +2505,23 @@ def _notify_get_recipients_groups(self, msg_vals=None):
         ``_notify_get_recipients_classify`` method.
         """
         return [
-            (
+            [
                 'user',
                 lambda pdata: pdata['type'] == 'user',
                 {'has_button_access': True}
-            ), (
+            ], [
                 'portal',
-                lambda pdata: pdata['type'] == 'portal',
+                lambda pdata: False and pdata['type'] == 'portal',  # activate only on demand if rights are enabled
                 {'has_button_access': False}
-            ), (
+            ], [
+                'follower',
+                lambda pdata: False and pdata['is_follower'],  # activate only on demand if rights are enabled
```

Comment:
seems louche

Reply:
Updated with an active flag on the group itself.